python/label_dict.hpp

```diff
@@ -48,11 +48,11 @@ struct label_dict_proxy {
         return locsets.size() + regions.size() + iexpressions.size();
     }
 
-    void import(const label_dict_proxy& other, std::string prefix = "") {
-        dict.import(other.dict, prefix);
-
+    auto extend(const label_dict_proxy& other, std::string prefix = "") {
```

Comment:
copy of `*this` intended when returning?

Reply:
no, good catch.